docs/en/guides/Plugin-test.md

```diff
@@ -647,9 +647,10 @@ elasticsearch 5.x | 3 | 142.15
 
 
 ### Workload 3
-#### Group 1 (2291.912s)
+#### Group 1 (3090.912s)
 scenario name | versions | elapsed time (sec)
 ---|---|---
+hystrix-scenario | 20 | 799.00
```

Comment:
place it in Workload1 Group2. Because the group1 also takes almost 3000s.

Reply:
According to `Plugin-test.md`, workload3 group1 is little faster

Reply:
Before I modified, According to `Plugin-test.md`, workload3 group1 is little faster

Reply:
Before I modified, According to `Plugin-test.md`, workload3 group1 is little faster

Reply:
Before I modified, According to Plugin-test.md, workload3 group1 is little faster than workload1 Group 2

Reply:
I see. But a workload is waiting for both groups to finish, two groups of workload take the same time will be better.

Reply:
if add to workload1 group2，total is：3090.98.  current workload3 group1， total is: 3090.912， is right？

Reply:
@aderm According to document, workload1 group2 is only 2300s. Much faster than 3000. Where is your data from? 

![image](https://user-images.githubusercontent.com/5441976/69125637-361cd500-0ae1-11ea-87fa-f9d55d0b7dbe.png)

Also verified from the CI process, https://builds.apache.org/blue/organizations/jenkins/skywalking-agent-test/detail/skywalking-agent-test/1297/pipeline/135

Reply:
OK, make sense. I am good.